SCANNER_VERSION

```diff
@@ -1 +1 @@
-2.37.3
+2.37.4
```

Comment:
May need to update this after merging https://github.com/stackrox/scanner/pull/2102

Reply:
Done.